src/components/header.tsx

```diff
@@ -1,14 +0,0 @@
-import * as React from "react";
-import { IBaseProps } from "./base";
-import { ClueHeaderComponent } from "../clue/components/clue-header";
-import "./header.sass";
```

Comment:
I believe this import was what was causing the right nav header to inherit a flex display which centered the section title text.  Fix is simple: in `right-nav.sass` we just need to add `display: flex` to the `header` class defined at line 104.

Reply:
Done